assets/app/scripts/controllers/pod.js

```diff
@@ -26,6 +26,12 @@ angular.module('openshiftConsole')
       }
     ];
 
+    // Check for a ?tab=<name> query param to allow linking directly to a tab.
+    if ($routeParams.tab) {
+      $scope.selectedTab = {};
```

Comment:
Each tab directive has a different scope value for `active`.

```
<tab heading="Details" active="selectedTab.details">
```

The expression has to be assignable as well, so you can't use `selectedTab === 'details'`.